Docs/source/usage/parameters.rst

```diff
@@ -242,6 +242,22 @@ If an electrostatic field solve is used the boundary potentials can also be set
 * ``boundary.reflect_all_velocities`` (`bool`) optional (default `false`)
     For a reflecting boundary condition, this flags whether the sign of only the normal velocity is changed or all velocities.
 
+.. _running-cpp-parameters-eb:
+
+Embedded Boundary Conditions
+----------------------------
+
+* ``warpx.eb_implicit_function`` (`string`)
+    A function of `x`, `y`, `z` that defines the surface of the embedded
+    boundary. That surface lies where the function value is 0 ;
+    the physics simulation area is where the function value is negative ;
+    the interior of the embeddded boundary is where the function value is positive.
+
+* ``warpx.eb_potential(t)`` (`string`)
```

Comment:
Once this is merged into development we'd like to add the picmi functionality to allow the embedded boundaries to be set up from a Python run script. I'm wondering if the `()` in the parameter name will cause a problem for this since setting it from picmi will be done through a call similar to `pywarpx.warpx.eb_potential(t) = xx` which will not be happy with the brackets in the parameter name. Is there a work around for this that I'm missing? Alternatively would it be okay to call it something like `warpx.eb_potential_t` instead?

Reply:
The parenthesis are Ok. In the picmi interface, this can be handled by using `setattr`. The code will look something like this:
`pywarpx.warpx.__setattr__('eb_potential(t)', xx)`.
You can see several examples of this in `Python/pywarpx/picmi.py`.

Reply:
Oh great, that is good to know. Thank you!